cluster-management/setup/other-settings/index.md

```diff
@@ -0,0 +1,115 @@
+gets loaded. Module auto-loading can be broken all-together if the
+operation requiring the module happens inside of a container. `iptables`
+and other netfilter features can easily encounter both of these issues.
+To force a module to be loaded early during boot simply list them in a
+file under `/etc/modules-load.d`. The file name must end in `.conf`.
+
+```sh
+echo nf_conntrack > /etc/modules-load.d/nf.conf
+```
+
+These files are processed early during the boot sequence. This means
+that updating `modules-load.d` via cloud-config will only take effect on
+the next boot unless the `systemd-modules-load` service is also
+restarted:
+
+```yaml
+#cloud-config
+
+write_files:
+  - path: /etc/modules-load.d/nf.conf
+    content: nf_conntrack
+
+coreos:
+  units:
+    - name: systemd-modules-load.service
+      command: restart
+```
+
+Further details can be found in the systemd man pages:
+[modules-load.d(5)](http://www.freedesktop.org/software/systemd/man/modules-load.d.html)
+[systemd-modules-load.service(8)](http://www.freedesktop.org/software/systemd/man/systemd-modules-load.service.html)
+
+
+## Tuning sysctl Parameters
+
+The Linux kernel offers a plethora of knobs under `/proc/sys` to control
+the availability of different features and tune performance parameters.
+For one-shot changes values can be written directly to the files under
+`/proc/sys` but persistent settings must be written to `/etc/sysctl.d`:
+
+```sh
+echo net.netfilter.nf_conntrack_max=131072 > /etc/sysctl.d/nf.conf
+sysctl --system
+```
+
+Some parameters, such as the conntrack one above, are only available
+after the module they control has been loaded. To ensure any modules are
+loaded in advance use `modules-load.d` as described above. A complete
+cloud-config using both would look like:
+
+```yaml
+#cloud-config
+
+write_files:
+  - path: /etc/modules-load.d/nf.conf
+    content: |
+      nf_conntrack
+  - path: /etc/sysctl.d/nf.conf
+    content: |
+      net.netfilter.nf_conntrack_max=131072
+
+coreos:
+  units:
+    - name: systemd-modules-load.service
+      command: restart
+    - name: systemd-sysctl.service
+      command: restart
+```
+
+Further details can be found in the systemd man pages:
+[sysctl.d(5)](http://www.freedesktop.org/software/systemd/man/sysctl.d.html)
+[systemd-sysctl.service(8)](http://www.freedesktop.org/software/systemd/man/systemd-sysctl.service.html)
+
+
+## Adding Custom Messages to MOTD
+
+By default when logging in interactively via SSH or a console a brief
+message reporting the CoreOS release channel, version, and a list of any
+services or other systemd units that have failed. Additional text can be
+added by dropping text files into `/etc/motd.d`. The directory may need
+to be created first and the file name must end in `.conf`.
+CoreOS 555.0.0 or later is required.
+
+```sh
+mkdir -p /etc/motd.d
+echo "This machine is dedicated to computing Pi" > /etc/motd.d/pi.conf
+```
+
+Or via cloud-config:
+```yaml
```

Comment:
Random FYI: not having a new line before the three backticks breaks our markdown parser.